portfolio/src/main/webapp/script.js

```diff
@@ -137,3 +148,141 @@ function deleteSpecificComment(comment) {
+      {
+        featureType: 'poi.park',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#447530'}]
+      },
+      {
+        featureType: 'road',
+        elementType: 'geometry',
+        stylers: [{color: '#f5f1e6'}]
+      },
+      {
+        featureType: 'road.arterial',
+        elementType: 'geometry',
+        stylers: [{color: '#fdfcf8'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry',
+        stylers: [{color: '#f8c967'}]
+      },
+      {
+        featureType: 'road.highway',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#e9bc62'}]
+      },
+      {
+        featureType: 'road.highway.controlled_access',
+        elementType: 'geometry',
+        stylers: [{color: '#e98d58'}]
+      },
+      {
+        featureType: 'road.highway.controlled_access',
+        elementType: 'geometry.stroke',
+        stylers: [{color: '#db8555'}]
+      },
+      {
+        featureType: 'road.local',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#806b63'}]
+      },
+      {
+        featureType: 'transit.line',
+        elementType: 'geometry',
+        stylers: [{color: '#dfd2ae'}]
+      },
+      {
+        featureType: 'transit.line',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#8f7d77'}]
+      },
+      {
+        featureType: 'transit.line',
+        elementType: 'labels.text.stroke',
+        stylers: [{color: '#ebe3cd'}]
+      },
+      {
+        featureType: 'transit.station',
+        elementType: 'geometry',
+        stylers: [{color: '#dfd2ae'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'geometry.fill',
+        stylers: [{color: '#b9d3c2'}]
+      },
+      {
+        featureType: 'water',
+        elementType: 'labels.text.fill',
+        stylers: [{color: '#92998d'}]
+      }]
+    }
+  );
+ 
+  addMarker({lat: 42.4550, lng: -76.4777}, "Freshman year dorm", iconBeachFlag, "<b>Mary Donlon Hall</b><br>Freshman year dorm - Chris lived on the fourth floor in a quad");
+  addMarker({lat: 42.4477, lng: -76.4853}, "Favorite library", iconBeachFlag, "<b>Uris Library</b><br>The underground is called the Cocktail lounge");
+  addMarker({lat: 42.4462, lng: -76.4823}, "Favorite dining hall", iconBeachFlag, "<b>Terrace dining hall</b><br>Has great food");
+  addMarker({lat: 42.4529, lng: -76.4774}, "Favorite gym", iconBeachFlag, "<b>Helen Newman Gym</b><br>Has gym, basketball courts, and swimming pool");
+}
+
+/** Creates a marker given a map, coordinates, description, and icon. */
+function addMarker(coordinates, title, icon, description){
+  const marker = new google.maps.Marker({
+    map: map,
+    animation: google.maps.Animation.DROP,
+    position: coordinates,
+    title: title,
+    icon: icon
+  });
+
+  const infoWindow = new google.maps.InfoWindow({content: description});
+  marker.addListener('click', () => {
+    infoWindow.open(map, marker);
+  });	
+}
```

Comment:
nit: Let's add a newline at the end of this file.

Reply:
fixed